features/export_test.go

```diff
@@ -30,8 +30,14 @@ func NumberOfFeatures() int {
 
 var FeaturesSupportedCallbacks = featuresSupportedCallbacks
 
+var FeatureNames = featureNames
```

Comment:
Tiny nitpick, from the usage below, this could be converted into a mock function instead to avoid mutating inplace.

Reply:
yes, it's better to mock this entirely rather than manipulate it directly in the test